portal-frontend/src/app/features/notifications/notification-details/parcel/parcel.component.html

```diff
@@ -5,7 +5,7 @@ <h3 class="flex-item">1. Identify Parcel(s) Under The Statutory Right of Way</h3
       <h4>Parcel {{ parcelInd + 1 }}</h4>
       <div>
         <button
-          *ngIf="showEdit"
+          *ngIf="showEdit && parcels.length > 1"
```

Comment:
This is inside a loop of parcels, so this code should only execute if there is at least one parcel. What does this fix?

Reply:
It's just one of the requirements:
> only show Edit Parcel button when there is more than one parcel

Reply:
This code wouldn't help with that requirement since it shouldn't render this section if there are no parcels. You should not add code that does not do anything.

Reply:
Ohhh, I read that as >  0, I see what you are saying.